Sources/PointFree/Episode/Show.swift

```diff
@@ -30,7 +30,7 @@ let episodeResponse =
           currentUser: currentUser,
           data: (permission, currentUser, subscriberState, episode),
           description: episode.blurb,
-          extraStyles: markdownBlockStyles <> pricingExtraStyles <> videoExtraStyles,
```

Comment:
I refactored the video styles a bit cause I needed something similar for images.